src/clr-addons/quick-list/quick-list.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="clr-form-control">
+  <div class="clr-row">
+    <div class="clr-col-12">
+      <ng-container *ngFor="let value of values; index as i; trackBy: trackById">
+        <clr-add-option [allValues]="allValues" [excludedValues]="values" (remove)="onRemoveValue(i)"
+                        [value]="value" (valueChanged)="onValueChanged($event, i)"
+                        [blankOption]="blankOption">
+        </clr-add-option>
+      </ng-container>
+      <div class="clr-row pt-0-5 pb-0-5">
```

Comment:
pb should not be used, as the next element needs to have a padding top if needed